### PR TITLE
Master

### PR DESCRIPTION
**Pull request by jeffreyjackson**

### CODE REVIEW

File: Anypic-iOS/Podfile [hunk 1]
```diff
@@ -2,6 +2,8 @@ source 'https://github.com/CocoaPods/Specs.git'
 
 workspace 'Anypic'
 
+use_frameworks!
+
```
Comment (nlutsenko) [inline Anypic-iOS/Podfile]: Maybe add platform + version?
Say `platform :ios, '8.0'`?

File: Anypic-iOS/Anypic/Images.xcassets/BackgroundLogin.imageset/Contents.json [hunk 2]
```diff
@@ -2,24 +2,17 @@
...
-      "scale" : "2x",
-      "filename" : "BackgroundLogin@2x.png"
+      "filename" : "BackgroundLogin@2x.png",
+      "scale" : "2x"
     },
     {
       "idiom" : "universal",
       "scale" : "3x"
-    },
-    {
```
Comment (nlutsenko) [inline Anypic-iOS/Anypic/Images.xcassets/BackgroundLogin.imageset/Contents.json]: Any chance to get context on this?

File: Anypic-iOS/Anypic/Images.xcassets/LaunchImage.launchimage/Contents.json [hunk 3]
```diff
@@ -72,22 +72,21 @@
...
-      "extent" : "full-screen",
       "filename" : "Default@2x-1.png",
+      "extent" : "full-screen",
       "scale" : "2x"
     },
     {
       "orientation" : "portrait",
       "idiom" : "iphone",
       "extent" : "full-screen",
-      "filename" : "Default-568h@2x-2.png",
```
Comment (nlutsenko) [inline Anypic-iOS/Anypic/Images.xcassets/LaunchImage.launchimage/Contents.json]: Any chance to get context on this change?

**Comment by nlutsenko** [comment]
Ok, this looks really awesome!
Few nits and questions...

Also, what do you think if we would split this into few pull requests?
Fixing images is probably not related to removing entitlements, as well as to using frameworks instead of static libraries from cocoapods?

**Comment by jeffreyjackson** [comment]
sure no problem, I can rework it all into separate pull requests later today if thats cool with you.  I can add some commentary on each PR for you as well.

**Comment by wellkunnected** [comment]
I can't ever seem to get this app to build with the "pods" folder being missing from the project. Everyone should know that if you updated to the latest version of Xcode, the old way of installing pods is not efficient. When I open a project in Xcode that requires the use of pods and there is no actual "pods" folder included in the project, I get an error message and underneath the project file there is a pods file but it's in red. Xcode wont build the app and will tell me to ensure I have cocoapods installed which clearly they are. The problem is is that the "pods" folder is not included with the project. i'm not sure what others are doing to get this to work but I can surely use some help!

**Pull request closed without merging**